core/include/mmcore/view/View3D.h

```diff
@@ -7,13 +7,23 @@
 
 #pragma once
 
-#include "mmcore/view/AbstractView3D.h"
+#include "mmcore/view/CallRenderView.h"
+#include "mmcore/view/BaseView.h"
+#include "mmcore/view/CameraControllers.h"
+#include "mmcore/view/CPUFramebuffer.h"
 
 namespace megamol {
 namespace core {
 namespace view {
 
-class MEGAMOLCORE_API View3D : public view::AbstractView3D {
+    inline constexpr auto cpu_fbo_resize = [](std::shared_ptr<CPUFramebuffer>& fbo, int width,
+                                              int height) -> void {
+        fbo->width = width;
+        fbo->height = height;
+        // TODO reallocate buffer?
```

Comment:
This look important?

Reply:
It sure does, but it should be based on how CPU framebuffers were handled in master at this point and I think Ospray for example allocates its own memory and then overwrites the contents of the CPUFramebuffer.
So right now, this only tries to at least let the framebuffer know its size so that this information can already be used while the actual memory is provided by the renderer...